### PR TITLE
chore: update lando to 77dad3a

### PR DESCRIPTION
**Pull request by sjinks**
## Description

Update `lando` to Automattic/lando-cli@77dad3a

This removes support for docker-compose v1 and adds support for the compose plugin.

## Pull request checklist

- [ ] Update [SETUP.md](https://github.com/Automattic/vip-cli/blob/trunk/docs/SETUP.md#list-of-environmental-variables) with any new environmental variables.
- [ ] Update [the documentation](https://github.com/Automattic/vip-cli/blob/trunk/docs).
- [x] [Manually test](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#manual-testing) the relevant changes.
- [ ] Follow the [pull request checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#new-pull-requests)
- [ ] Add/update [automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) as needed.

## New release checklist

- [ ] [Automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) pass.
- [ ] The [Preparing for release checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#preparing-for-release) is completed.

## Steps to Test

E2E tests should pass.

### CODE REVIEW

File: src/lib/dev-environment/dev-environment-lando.ts [hunk 1]
```diff
@@ -693,28 +696,38 @@ async function ensureNoOrphantProxyContainer( lando: Lando ): Promise< void > {
...
+		engine: string;
+		composePlugin: string;
+		compose: string;
+	};
+
+	lando.log.verbose( 'docker-engine version: %s', engine );
+	if ( ! engine ) {
+		if ( ! lando.config.dockerBin ) {
+			throw new UserError(
+				'docker could not be located! Please follow the following instructions to install it - https://docs.docker.com/engine/install/'
```
Comment (rinatkhaziev) [inline src/lib/dev-environment/dev-environment-lando.ts]:
```suggestion
				'docker binary could not be located! Please follow the following instructions to install it - https://docs.docker.com/engine/install/'
```

File: src/lib/dev-environment/dev-environment-lando.ts [hunk 2]
```diff
@@ -693,28 +696,38 @@ async function ensureNoOrphantProxyContainer( lando: Lando ): Promise< void > {
...
 			'Failed to connect to Docker. Please verify that Docker engine (service) is running and follow the troubleshooting instructions for your platform.'
 		);
 	}
+	lando.log.verbose( 'docker-compose version: %s', compose );
+	lando.log.verbose( 'compose plugin version: %s', composePlugin );
+
+	if ( ! composePlugin ) {
+		if ( ! compose ) {
+			throw new Error(
+				'docker-compose could not be located! Please follow the following instructions to install it - https://docs.docker.com/compose/install/'
```
Comment (rinatkhaziev) [inline src/lib/dev-environment/dev-environment-lando.ts]:
```suggestion
				'docker-compose binary could not be located! Please follow the following instructions to install it - https://docs.docker.com/compose/install/'
```

**Review by rinatkhaziev — APPROVED** [review]

File: src/lib/dev-environment/dev-environment-lando.ts (hunk 1 above)
Comment (sjinks) [reply src/lib/dev-environment/dev-environment-lando.ts]: And "follow the following" :-)

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@automattic/vip",
-  "version": "2.39.8-dev.0",
+  "version": "2.40.0-dev.4",
```
Comment (sjinks) [inline package.json]: ⚠️ Revert this before merging to `trunk`❗

Comment (sjinks) [reply package.json]: ✅

**Pull request merged**